src/sinol_make/commands/init/__init__.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import os
+import subprocess
+import tempfile
+
+from sinol_make import util
+from sinol_make.interfaces.BaseCommand import BaseCommand
+
+
+class Command(BaseCommand):
+    """
+    Class for "init"
+    """
+
+    def get_name(self):
+        return "init"
+
+    def configure_subparser(self, subparser: argparse.ArgumentParser):
+        parser = subparser.add_parser(
+            self.get_name(),
+            help='Create package from the template',
+            description='Create package from predefined template with given id.'
+        )
+        parser.add_argument('task_id', type=str, help='Id of the task to create')
+
+    def download_template(self):
+        repo = 'https://github.com/sio2project/sinol-make.git'
+        package_dir = 'sinol-make/example_package'
+        self.used_tmpdir = tempfile.TemporaryDirectory()
+        tmp_dir = self.used_tmpdir.name
+        ret = subprocess.run(['git', 'clone', '-q', '--depth', '1', repo], cwd=tmp_dir)
+        if ret.returncode != 0:
+            util.exit_with_error("Could not access repository. Please try again.")
+        return os.path.join(tmp_dir, package_dir)
+
+    def move_folder(self):
+        mapping = {}
+        mapping[self.template_dir] = os.getcwd()
+        for root, dirs, files in os.walk(self.template_dir):
+            for directory in dirs:
+                mapping[os.path.join(root, directory)] = os.path.join(mapping[root], directory)
+                os.mkdir(os.path.join(mapping[root], directory))
+            for file in files:
+                dest_filename = file
+                if file[:3] == 'abc':
+                    dest_filename = self.task_id + file[3:]
+                os.replace(os.path.join(root, file), os.path.join(mapping[root], dest_filename))
+
+    def update_config(self):
+        with open(os.path.join(os.getcwd(), 'config.yml')) as config:
+            config_data = config.read()
+        config_data = config_data.replace('sinol_task_id: abc', f'sinol_task_id: {self.task_id}')
+
+        with open(os.path.join(os.getcwd(), 'config.yml'), 'w') as config:
+            config.write(config_data)
+
+    def run(self, args: argparse.Namespace):
+        if len(args.task_id) != 3:
+            util.exit_with_error("Invalid task id. Task id should be 3 characters long.")
```

Comment:
I don't think we should limit the task id to exactly 3 characters.

Reply:
Task id is already required to be exactly 3 characters long:
https://github.com/sio2project/sinol-make/blob/0ed77a26199ddfebf70bb33105d221c46cbe9353/src/sinol_make/helpers/package_util.py#L20-L23

Reply:
That's only true if `sinol_task_id` is not set in config. `sinol_task_id` can be any length

Reply:
Good point. Sometimes understanding read code/text is quite hard 😅